node/_errors.ts

```diff
@@ -83,15 +83,47 @@ export class ERR_INVALID_ARG_TYPE extends NodeTypeError {
   constructor(a1: string, a2: string | string[], a3: unknown) {
     super(
       "ERR_INVALID_ARG_TYPE",
+      // TODO(kt3k): Expression for a2 is wrong.
```

Comment:
Wrong how? Do you mean it differs from node's? If so, please mention that explicitly. :-)